template/components/ShareButton.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <a :href="shareURL" @click.prevent="share()" target="_blank"
+     :class="[ isButton ? `btn btn-${networkName}` : '' ]">
+    <slot>
```

Comment:
The default slot currently has the longest possible share button, but we could change this.